doc/nrf/libraries/others/event_manager_proxy.rst

```diff
@@ -0,0 +1,162 @@
+.. _event_manager_proxy:
+
+Event Manager Proxy
+###################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Event Manager Proxy is a library that passes Event Manager events between cores.
+It connects two separate instances on different cores passing registered messages using IPC Service.
```

Comment:
``` suggestion
The Event Manager Proxy is a library that allows to exchange Event Manager's events between cores.
It connects two separate instances of Event Manager with use  of  IPC service.
```

I think `registered messages` is implementation detail. Let's keep it at minimum.